Simplenote/Resources/release_notes.txt

```diff
@@ -1,2 +1 @@
--   Some bug fixes
-
+- Some bug fixes
```

Comment:
Looking at the `.pot`, I wonder why what looks like the same string in resulted in different formatting 🤔 The diff in this file is the answer 😄